updatecli/scripts/check-jdk.sh

```diff
@@ -45,8 +56,12 @@ case "${1}" in
     platforms=("x64_linux" "x64_windows" "aarch64_linux" "s390x_linux");;
   17.*+*)
     platforms=("x64_linux" "x64_windows" "aarch64_linux" "s390x_linux");;
-  21*+*)
+  19.*+*)
```

Comment:
jdk19 ?

Reply:
Good catch! Removed in https://github.com/jenkins-infra/jenkins-infra/pull/3113/commits/072d4dc614a15f04c7c1f4db5ed6c160fab17950